components/sso-saml/org.wso2.carbon.identity.sso.saml/pom.xml

```diff
@@ -108,6 +108,7 @@
                             org.wso2.carbon.core.*,
                             org.wso2.carbon.registry.core.*,
                             org.wso2.carbon.security.keystore.*,
+                            org.apache.commons.lang.StringUtils,
```

Comment:
This is already fixed
https://github.com/wso2/carbon-identity/edit/master/components/sso-saml/org.wso2.carbon.identity.sso.saml/pom.xml